osbs/api.py

```diff
@@ -40,6 +49,9 @@ def catch_exceptions(*args, **kwargs):
         except OsbsException:
             # Re-raise OsbsExceptions
             raise
+        except (MethodCallError, MethodSignatureError, CallOrderError):
```

Comment:
Instead of actually importing these, can we leverage their `__module__` attribute, and check it in the `Exception` block below?

```python
>>> MethodSignatureError.__module__
'flexmock'
>>> CallOrderError.__module__
'flexmock'
>>> MethodCallError.__module__
'flexmock'
```